portfolio/index.html

```diff
@@ -0,0 +1,137 @@
+        for others that I will work on shortly.
+        <br>
+      </h3>
+    <div class="container-fluid">
+      <div class="container row">
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832585/nb_lsoysf.png" alt="Native Brews">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href="https://codepen.io/artismarti/full/akrZoB/">Native Brews</a></h4>
+              <p>Indian alcobev startup.</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832584/ada_lxftp1.png" alt="Ada Tribute">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href="https://codepen.io/artismarti/full/akrZoB/">Ada Lovelace</a></h4>
+              <p>FreeCodeCamp Tribute Page</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832585/meta_ikn0wc.png" alt="Portfolio">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href="https://codepen.io/artismarti/full/akrZoB/">Portfolio</a></h4>
+              <p>FreeCodeCamp Portfolio Page</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832584/random_quote_pgspcu.jpg" alt="Quote Generator">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href="#">Quote Generator</a></h4>
+              <p>FreeCodeCamp Quote Generator</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832584/weather_paois9.png" alt="Weather">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href="#">Weather App</a></h4>
+              <p>FreeCodeCamp Weather App</p>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-2 col-sm-12">
+          <div class="thumbnail">
+            <img class="port_img" src="http://res.cloudinary.com/artismarti/image/upload/v1474832584/wiki_drnvs3.png" alt="Wikipedia Viewer">
+            <div class="caption">
+              <h4><a class="portfolio" target="_blank" href"#">Wikipedia Viewer</a></h4>
+              <p>FreeCodeCamp Wikipedia Viewer</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="pageFour" id="contact">
```

Comment:
I think the nesting is a bit off here. A good way to check is to use code folding in your editor or codepen and make sure you closed off all the previous divs.